Combobox.js

```diff
@@ -669,8 +669,11 @@ define([
 				// inputNode does not contain text
 				if (!this.hasDownArrow) {
 					// in auto complete mode
-					this.closeDropDown();
-					this._toggleComboPopupList();
+					if (this._isMobile) {
+						this._toggleComboPopupList();
+					} else {
+						this.closeDropDown();
+					}
```

Comment:
You should have a test and a ticket for the bug you are fixing.  I also don't understand the commit comment.  It says:

> when `inputNode` was empty and in mobile, the popup used to close by itself

That makes it sound like ComboPopup would close immediately after opening, since when the ComboPopup opens the `inputNode` is empty.  Actually, I'm not sure if inputNode is the `<input>` in the original ComboBox or the `<input>` in the ComboPopup.